README.md

```diff
@@ -10,6 +10,9 @@
   $ sudo apt-get install python3-pip
 4. install flask for python3
   $ sudo pip3 install flask
+Error
```

Comment:
Please create a separate section for Errors last in the document, such as ##Errors. And make sure to separate it with one newline from the above section (below the line "$ ./run.py").